nomad-aws/examples/irsa/main.tf

```diff
@@ -3,7 +3,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = "~>3.0"
```

Comment:
This was picked up by `terraform validate` as not being correct. I copied the version used in other examples.